code/clue.js

```diff
@@ -10,50 +9,202 @@ const mrGreen = {
   occupation: 'Entrepreneur'
 }
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+const professorPlum = {
+  firstName: 'Victor',
+  lastName: 'Plum',
+  color: 'purple',
+  description: 'He is very quiet',
+  age: 39,
+  image: 'assets/plum.png',
+  occupation: 'Professor'
+}
+
+const missScarlet = {
+  firstName: 'Cassandra',
+  lastName: 'Scarlet',
+  color: 'red',
+  description: 'She is very smart',
+  age: 37,
+  image: 'assets/scarlet.png',
+  occupation: 'Lawyer'
+}
+
+const mrsPeacock = {
+  firstName: 'Eleanor',
+  lastName: 'Peacock',
+  color: 'blue',
+  description: 'She is very rich',
+  age: 58,
+  image: 'assets/peacock.png',
+  occupation: 'Doctor'
+}
+
+const colonelMustard = {
+  firstName: 'Jack',
+  lastName: 'Mustard',
+  color: 'yellow',
+  description: 'He is very loyal',
+  age: 62,
+  image: 'assets/mustard.png',
+  occupation: 'Colonel'
+}
+
+const mrsWhite = {
+  firstName: 'Mrs',
+  lastName: 'White',
+  color: 'white',
+  description: 'She is very kind',
+  age: 65,
+  image: 'assets/white.png',
+  occupation: 'Actress'
```

Comment:
Clever and fun descriptions of all the suspects 😄 I love the creativity in this project! One tiny little comment would be to separate the objects for suspects, from objects for weapons by a comment //